### PR TITLE
Upgrade Inventory Page

### PR DESCRIPTION
**Pull request by aqsawaheed-8840**
Upgrade inventoy page into new inventory page

Inventory

### CODE REVIEW

File: aqsa.md [hunk 1]
```diff
@@ -15,3 +15,13 @@ The screen consists of three blocks:
...
+
+## ISSUE # 216
+**Upgrade Inventory Page**
+
+It displays a table which has the informaion about all the products. 
+
+The table can be filtered based on different stores. 
+
+It also allows the user to search via product name, or barcode. 
+
```
Comment (ammarhaider001) [inline aqsa.md]: remove this file

File: src/components/admin/Inventory.vue [hunk 2]
```diff
@@ -1,135 +1,204 @@
...
+    Paginator,
   },
-  data () {
+  data() {
     return {
-      search: '',
-      company: ''
-    }
+      checkedValue: -1,
+      filter: [1,0,0,0],
```
Comment (ammarhaider001) [inline src/components/admin/Inventory.vue]: remove filter if it is not used.

**Comment by ammarhaider001** [comment]
also remove all the console.log statements

File: src/components/common-components/Paginator.vue [hunk 3]
```diff
@@ -1,12 +1,14 @@
 <template>
     <div class="pagination">
-        <a href="#" @click="moveLeft"><strong>&lt;&lt;</strong></a>
```
Comment (ammarhaider001) [inline src/components/common-components/Paginator.vue:3]: dont change this implementation just change the styling.

File: src/components/admin/Inventory.vue [hunk 4]
```diff
@@ -1,135 +1,204 @@
 <template>
-  <div class="diff-shadow">
```
Comment (ammarhaider001) [inline src/components/admin/Inventory.vue:2]: don't remove any existing code. comment out the existing code instead. and mention
// Existing Code Starts Here
// ...
// Existing Code Ends Here

File: src/components/admin/Inventory.vue [hunk 5]
```diff
@@ -1,135 +1,204 @@
...
+      userdata: "getUser",
+    }),
   },
   methods: {
-
+    onCheckChange: async function(e: any){
+      this.checkedValue = parseInt(e.target.value)
+      
+      console.log(this.company)
+      console.log(this.search)
```
Comment (ammarhaider001) [inline src/components/admin/Inventory.vue]: remove these console logs

**Review by ammarhaider001 — CHANGES_REQUESTED** [review]

**Review by ammarhaiderak — APPROVED** [review]

**Pull request merged**